terraform/cloud-platform-components/cert-manager.tf

```diff
@@ -23,9 +23,8 @@ data "aws_iam_policy_document" "cert_manager" {
     actions = ["route53:ChangeResourceRecordSets"]
 
     resources = ["${compact(list(
-      "arn:aws:route53:::hostedzone/${data.terraform_remote_state.cluster.hosted_zone_id}",
-      "${terraform.workspace == local.live_workspace ? format("%s/%s", "arn:aws:route53:::hostedzone", data.terraform_remote_state.global.cp_zone_id) : ""}",
-    ))}"]
+      "${terraform.workspace == local.live_workspace ? "arn:aws:route53:::hostedzone/*" : format("%s/%s", "arn:aws:route53:::hostedzone", data.terraform_remote_state.cluster.hosted_zone_id)}",
```

Comment:
What if we re-wrote this like so:
`"${format("arn:aws:route53:::hostedzone/%s", terraform.workspace == local.live_workspace ? "*" : data.terraform_remote_state.cluster.hosted_zone_id}"`
I think it's much easier to read? (haven't tested syntax validity!)

Reply:
updated the "aws_iam_role" "external_dns" resources filter as suggested